### ISSUE TITLE
Disabling accessibility options has no effect

### ISSUE THREAD

**Issue by gilluminate** [highcharts/highcharts#7329]
Setting the accessibility option to `enabled: false` and Setting the keayboardNavigation option to `enabled: false` seems to have no effect whatsoever. Even continues to include instructions about how to interact with the chart ("Use up and down arrows to interact with most screen readers") and continues to allow for keyboard interaction.

#### Api references:
https://api.highcharts.com/highcharts/accessibility.enabled
https://api.highcharts.com/highcharts/accessibility.keyboardNavigation.enabled

#### Live demo with steps to reproduce
http://jsfiddle.net/jgill/fumea44q

**Comment by pawelfus** [comment]
Hi @gilluminate 

Accessibility options should be set directly in the [config](https://api.highcharts.com/highcharts/accessibility), not under `chart` options:

```
Highcharts.chart('container', {
  accessibility: {
    enabled: false,
    keyboardInteraction: {
      enabled: false
    }
  },
  ...
});
```

**Comment by gilluminate** [comment]
👍  My bad. Thanks.
